TestFoundation/TestFileManager.swift

```diff
@@ -1565,15 +1557,15 @@ VIDEOS=StopgapVideos
             ("test_displayNames", test_displayNames),
         ]
         
-        #if !DEPLOYMENT_RUNTIME_OBJC && NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT
+        #if !DEPLOYMENT_RUNTIME_OBJC && NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT && !os(Android)
```

Comment:
Ditto, NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT not set, so checking for Android is redundant.

Reply:
Same explanation as above.